vars/loadResources.groovy

```diff
@@ -1,9 +1,12 @@
+import io.openshift.Utils
+
 def call(Map args = [:]) {
-  if(!args.file) {
+  if (!args.file) {
     error "Missing manadatory parameter: file"
   }
 
   def yaml = readYaml(file: args.file)
```

Comment:
@pradeepitm12 would it be better to move validation call before `readYaml`?

Reply:
Yes, we can validate first and then load it.